packages/cln_plugin/lib/src/rpcmethods/manifest/getmanifest.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:cln_plugin/src/rpcmethods/abstractrpcmethod.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/option.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/feature.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/notification.dart';
+
+class ManifestMethod extends AbstractRpcMethod {
+  late List<AbstractRpcMethod> _rpcMethods;
+  late List<String> _subscriptions;
+  late List<Option> _options;
+  late List<String> _hooks;
+  late Feature _features;
+  late bool _dynamic = false;
+  late Set<Notification> _notifications;
+
+  ManifestMethod(String name, String usage, String description)
+      : super("getmanifest", "", "");
+
+  void addFeature(String node, String channel, String init, String invoice) {
+    _features = Feature(node, channel, init, invoice);
+  }
+
+  void addMethods(List<AbstractRpcMethod> methods) {
+    _rpcMethods.addAll(methods);
+  }
+
+  void addMethod(AbstractRpcMethod method) {
+    _rpcMethods.add(method);
+  }
+
+  void addOption(Option option) {
+    _options.add(option);
+  }
+
+  void addSubscriptions(String subscription) {
+    _subscriptions.add(subscription);
+  }
+
+  void addHook(String hook) {
+    _hooks.add(hook);
+  }
```

Comment:
```suggestion
  void addOption(Option option) {
    _options.add(option);
  }

  void addSubscriptions(String subscription) {
    _subscriptions.add(subscription);
  }

  void addHook(String hook) {
    _hooks.add(hook);
  }
```

all the add can stay there for the moment :)